### DISCUSSION TITLE
which access scheme is used to transmit?

### DISCUSSION THREAD

**Discussion by michrft** [peterantypas/maiana#104] [General]
Hello and many thanks for this huge work.

I am at the very beginning of the project so I only parsed your code.
I did not find anything about TDMA.
Which access scheme is used to transmit?
I saw AIS message 15, 18 and 24. These messages do not use the same access code (except ITDMA which is specific).

Best regards
Mich

**Comment by matthew-quirke** [comment]
CSMA
